realm/realm-library/src/main/java/io/realm/RealmFieldType.java

```diff
@@ -42,17 +42,16 @@
     FLOAT(9),
     DOUBLE(10),
     OBJECT(12),
-    LIST(13);
-    // BACKLINK(14); Not exposed until needed.
+    LIST(13),
+    BACKLINK(14);
```

Comment:
technically it's a backlink that points to the linking objects (afaik) but it's called `linking objects` everywhere else.